src/components/App.js

```diff
@@ -1,44 +1,16 @@
-import { books } from '../material/bookdata.json'
-import BookCard from './BookCard'
+import BookCardsPage from './BookCardsPage'
 import styled from 'styled-components/macro'
-import React, { useState } from 'react'
 
 export default function App() {
-  const [bookmarkedBooks, setBookmarkedBooks] = useState([])
-
-  let bookmarkedBooksArray
-  function handleBookmarkClick(currentBook) {
-    if (bookmarkedBooks.includes(currentBook)) {
-      bookmarkedBooksArray = bookmarkedBooks.filter(
-        book => book !== currentBook
-      )
-    } else {
-      bookmarkedBooksArray = [...bookmarkedBooks, currentBook]
-    }
-    setBookmarkedBooks(bookmarkedBooksArray)
-  }
-
   return (
     <>
       <AppLayout>
-        {books.map(card => (
-          <BookCard
-            key={card.id}
-            cover={card.cover}
-            title={card.title}
-            author={card.author}
-            description={card.content}
-            onBookmarkClick={handleBookmarkClick}
-            bookmarkedBooks={bookmarkedBooks}
-          />
-        ))}
+        <BookCardsPage />
       </AppLayout>
     </>
   )
 }
 
 const AppLayout = styled.div`
-  display: grid;
-  gap: 0.2%;
-  padding: 1%;
+  margin: 0;
```

Comment:
Isnt this the default value for a div? So if there are no globalstyles for all divs, which should be avoided (?), you could remove the margin here.

Reply:
I think so too.

Reply:
Thanks, i removed the margin here